br/pkg/lightning/restore/restore.go

```diff
@@ -2137,13 +2150,45 @@ type deliverResult struct {
 	err      error
 }
 
+func (cr *chunkRestore) adjustRowID(rowID int64, rc *Controller) (int64, error) {
+	if rowID > cr.originalRowIDMax {
+		if cr.curRowIDBase == 0 || cr.curRowIDBase > cr.curRowIDMax {
```

Comment:
the allowed rowid range is [base, max]? maybe add some comments on the field declare part.